omod/src/main/java/org/openmrs/module/dataintegrity/page/controller/ResultsPageController.java

```diff
@@ -32,7 +32,7 @@ public void get(PageModel model,
 		}
 		if (ruleId == null || ruleId.equals("")) {
 			model.addAttribute("results", dataintegrityService.getAllResults());
-			model.addAttribute("rule", null);
+			model.addAttribute("rules", dataintegrityService.getAllRules());
```

Comment:
1. The rule is to show the details of just one of the rules. 

2. You need to put the rules outside the check so that they are always available as a filter and probably rename the variable to all rules